indexes/social.js

```diff
@@ -2,7 +2,8 @@ const bipf = require('bipf')
 const pl = require('pull-level')
 const pull = require('pull-stream')
 const Plugin = require('./plugin')
-const { offsets } = require('../operators')
+const json = require('level-codec/lib/encodings').json
```

Comment:
This was the problem @staltz ;-)

Reply:
level understands 'json' fine but level-post does not :(

Reply:
Aahhh I see now